gitversion.yml

```diff
@@ -25,6 +25,10 @@ branches:
     tag: proj-{BranchName}
     regex: projects/(.*?)
     source-branches: ['master']
+  feature:
+    tag: feature.{BranchName}
```

Comment:
tag for project branches is proj-{branchname}, let's keep the same pattern ? I'm not entirely sure on top of my head but I think there's a reason for not using a dot. (for release pipelines).

I would suggest feature-{BranchName} instead of feature.{BranchName}

Reply:
Actually I preferred to follow the `dev` pattern (with a dot) as we don't have any `proj` for Uno yet